### DISCUSSION TITLE
Remote calendar not syncing with Day planner

### DISCUSSION THREAD

**Discussion by YunkaiY** [ivan-lednev/obsidian-day-planner#411] [Q&A]
Hi, I saw your great plugin of Obsidian, and was trying to add my calendars from iCloud and outlook. I have already added the links from iCloud and outlook as suggested in your instructions, no error was shown, yet there seemed to be no syncing with my calendars. I tried to click the `manually sync with all remote calendars` button, and restart both Obsidian and my laptop, but nothing changed. I am wondering how I can solve this problem.

I am putting the format of links I exported from all my calendars down below for your reference.
<img width="300" alt="Screenshot 2024-04-03 at 14 44 07" src="https://github.com/ivan-lednev/obsidian-day-planner/assets/151228190/57ecfeb8-23dc-4b84-9a2a-3b7a4ea308a2">

**Comment by Juneddo** [comment]
same with my situdation :(

**Comment by ivan-lednev** [comment]
Seems like it's related to #587

**Comment by pentecosthub** [comment]
Maybe I am slow but I can't get my google calendars to sync either.
